src/js/renderShabad.js

```diff
@@ -1,33 +1,34 @@
 function renderShabad(gurbani, nav) {
   document.body.classList.remove("loading");
-  let footnav = '';
+
+  let footnav = null;
   if(typeof nav != "undefined") {
     let link        = navLink(nav);
     let pagination  = [];
     if (typeof nav.previous != "undefined") {
-      pagination.push(h('a', { href: link + nav.previous }, '&laquo;'));
+      pagination.push(<a href={link + nav.previous}>&laquo;</a>);
```

Comment:
Does this work? Or do you need to use template literals?

Reply:
Yeah this will work ! :D